src/gui/main.cpp

```diff
@@ -13,22 +13,27 @@
 
 class MyApplication : public QApplication {
 public:
-    MyApplication(int argc, char* argv[]):
+    // Pass argc by reference
+    // Otherwise we give reference to stack variable
+    // Which results in memory error (segfault)
+    // When breaking Qt's  main loop
+    // (e.g. exec() of dialogs or message boxes)
```

Comment:
Please convert these multiple phrases to a sentence: add a period, do not start new lines from upper-case.